tab-sessions/info.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Tab Sessions",
+  "identifier": "tab-sessions",
+  "script": "tab-sessions.qml",
+  "authors": ["@macsunmood"],
+  "platforms": ["linux", "macos", "windows"],
+  "version": "0.0.1",
+  "minAppVersion": "22.8.4",
+  "description" : "This script lets you manage <b>Tab Sessions</b> - predefined layouts of tabs to be opened. \n\nTo create a Tab Session make a tag named `[[<title>]]` and the script will automatically rebuild the sessions on load."
```

Comment:
`<title>` will be interpreted as (unknown) opening tag in html and so the whole text after it will be ignored.
Please try `&lt;title&gt;` instead.

Reply:
And can you please explain what `title` you are talking about.

Reply:
@pbek solved in new PR.